src/masoniteorm/query/QueryBuilder.py

```diff
@@ -1007,6 +1007,22 @@ def first(self, query=False):
 
         return self.prepare_result(result)
 
+    def last(self, primary_key=None, query=False):
+        """Gets the last record.
+
+        Returns:
+            dictionary -- Returns a dictionary of results.
+        """
+        _primary_key = primary_key if primary_key else self._model.get_primary_key()
```

Comment:
I would actually change the naming of this to `column` because there will be times you want to get the last element based on a column. Like the highest price and do something like `last('price')`

Reply:
Yes makes sense ! ✅